.gitignore

```diff
@@ -40,3 +40,5 @@ yarn-debug.log*
 
 # Ignore IntelliJ
 .idea
+
+.DS_Store
```

Comment:
fyi both you and @bradweee have this: https://github.com/carrot-u/carrot-u-admin-app/pull/13/files#diff-a084b794bc0759e7a6b77810e01874f2R45

first person to merge gets to avoid the merge conflict :)